src/webgpu/api/validation/shader_module/entry_point.spec.ts

```diff
@@ -43,23 +48,52 @@ const kEntryPointTestCases = [
 g.test('compute')
   .desc(
     `
-Tests calling createComputePipeline(Async) with valid vertex stage shader and different entryPoints,
+Tests calling createComputePipeline(Async) with valid compute stage shader and different entryPoints,
 and check that the APIs only accept matching entryPoint.
 `
   )
-  .params(u => u.combine('isAsync', [true, false]).combineWithParams(kEntryPointTestCases))
+  .params(u =>
+    u
+      .combine('isAsync', [true, false])
+      .combine('shaderModuleStage', ['compute', 'vertex', 'fragment'] as TShaderStage[])
+      .beginSubcases()
+      .combine('provideEntryPoint', [true, false])
+      .combine('extraEntryPoint', [true, false])
```

Comment:
We also need to test different stages for the extraEntryPoint. For example a shader module with one vertex and one fragment stage should still be usable with default entry points.

This might be best as a separate test though, to avoid overparameterizing these tests for little extra coverage.

Reply:
Good catch!